Sample/SlimDownYourAggregate/SlimDownYourAggregate/Aggregate.cs

```diff
@@ -0,0 +1,83 @@
+namespace SlimDownYourAggregate;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class ChemicalReaction
+{
+    public Guid Id { get; private set; }
```

Comment:
why do reactions need to know their identity todo their job. :P

Reply:
Ask Chat GPT :) That's the initial state :) See more: https://www.architecture-weekly.com/p/webinar-8-slim-down-your-aggregates (the first 20m are without a paywall).

Reply:
Check refactored (and not compiling yet) solution, it’s removed there😉